extensions/binding/org.eclipse.smarthome.binding.hue.test/src/test/java/org/eclipse/smarthome/binding/hue/internal/LightStateConverterOSGiTest.java

```diff
@@ -29,13 +32,80 @@
 public class LightStateConverterOSGiTest extends JavaOSGiTest {
```

Comment:
Luckily nothing in this test needs extra support, so you can drop inheritance completely.

Reply:
And rename the test class accordingly, tnx.

Reply:
Yes, great. I removed it.